fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/protocol/ProtocolMetric.java

```diff
@@ -3,40 +3,76 @@
 package com.yahoo.bard.webservice.data.metric.protocol;
 
 import com.yahoo.bard.webservice.data.metric.LogicalMetric;
+import com.yahoo.bard.webservice.data.metric.LogicalMetricInfo;
 
 import java.util.Map;
 
 /**
- * Protocol metrics are capable of handling transforming signals from the client.
+ * <p>Protocol metrics are metrics that have the ability to have transformations applied to them at query time. Protocol
+ * metrics themselves represent valid metrics even if no calculations are applied.
```

Comment:
'Protocol metrics should generally represent valid metrics even if no calculations are applied.'